generators/Program.fs

```diff
@@ -49,9 +49,11 @@ let private regenerateTestClasses options =
     
     createExercises options
     |> List.filter (shouldBeIncluded options)
-    |> List.iter regenerateTestClass'
-
-    Log.Information("Re-generated test classes.")
+    |> function
+        | [] -> Log.Error "No exercises matched given options."
+        | x ->
```

Comment:
Tiny nit: could you rename `x` to `exercises`?

Reply:
Good idea! I could stand to develop a habit of naming those better.